test/extended/networking/services.go

```diff
@@ -0,0 +1,79 @@
+package networking
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/test/e2e"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[networking] services", func() {
+	f1 := e2e.NewFramework("net-isolation1")
+	f2 := e2e.NewFramework("net-isolation2")
+
+	It("should allow connections to another pod on the same node via a service IP", func() {
+		Expect(checkService(f1, f1, false)).To(Succeed())
+	})
+
+	It("should allow connections to another pod on a different node via a service IP", func() {
+		Expect(checkService(f1, f1, true)).To(Succeed())
+	})
+
+	It("should allow connections to another pod in a different namespace on the same node via a service IP", func() {
+		if pluginIsolatesNamespaces() {
```

Comment:
Same suggestion regarding using a variable in place of calling this function in each test.